src/main/kotlin/dev/arbjerg/ukulele/command/HelpCommand.kt

```diff
@@ -1,15 +1,35 @@
 package dev.arbjerg.ukulele.command
 
+import dev.arbjerg.ukulele.config.BotProps
 import dev.arbjerg.ukulele.features.HelpContext
 import dev.arbjerg.ukulele.jda.Command
 import dev.arbjerg.ukulele.jda.CommandContext
+import net.dv8tion.jda.api.MessageBuilder
 import org.springframework.stereotype.Component
 
 @Component
 class HelpCommand : Command("help") {
-    override suspend fun CommandContext.invoke() {
-        val command = beans.commandManager[argumentText.trim()] ?: return replyHelp()
-        replyHelp(command)
+    override suspend fun CommandContext.invoke() = when {
```

Comment:
When you have a `when()` block like this it makes more sense to just have an `if/else` statement. `when` is functionally the same but an `if/else` looks more readable here since you only have two cases.

Reply:
Done